project/build.scala

```diff
@@ -69,7 +70,8 @@ object ScalatraBuild extends Build {
     base = file("core"),
     settings = scalatraSettings ++ Seq(
       libraryDependencies <++= scalaVersion(sv => {
-        val defau = Seq(servletApi % "provided;test",
+        val default = Seq(servletApi % "provided;test",
+          reflect(sv),
```

Comment:
It's been a while since I've been into macros.  Is this needed in Scala 2.11, or just 2.10?  This, unlike most other libraries, isn't binary compatible across patch versions.  Since 2.10 is no longer maintained, forcing 2.10.5 is not such a big deal.  This would be unfortunate in 2.11.x.